src/Service/DynamoDb/src/Result/BatchGetItemOutput.php

```diff
@@ -169,7 +169,7 @@ private function populateResultBatchGetRequestMap(array $json): array
     }
 
     /**
-     * @return array<string, array>
+     * @return array<string, array<string, AttributeValue>[]>
```

Comment:
Shouldn't it be?

```suggestion
     * @return array<string, array<string, AttributeValue>>
```

Reply:
given `GetItemOutput` returns a list of map, i think it's OK to says `BatchGetItemOutput` returns a map of list of map...